tools/docker/installation.md

```diff
@@ -0,0 +1,89 @@
+---
+title: Docker
+page: Installation
+section: tools
+---
+
+#**Getting started with Docker**
+
+###**Installing Docker on Fedora**
+
+####**Prerequisites**
+
+Docker is currently supported on the following versions of Fedora:
+
+* Fedora 20
+* Fedora 21
+* Fedora 22
+
+Docker requires a **64-bit** version of Fedora and a minimum kernel version of **3.10**. 
+
+The following command displays your kernel version and architecture:
+
+```
+$ uname -r
+4.0.8-300.fc22.x86_64
+```
+
+If your kernel version is less than 3.10, you can update it using the following command:
+
+```
+$ sudo dnf update kernel
+```
+
+You need to restart your computer to boot with the new kernel version.
+
+####**Installation**
+
+The easiest way to install Docker on Fedora is from a Fedora package:
+
+```
+$ sudo dnf install docker
```

Comment:
Since you mentioned F2{0,1,2}, it would be worth saying, that on F2{0,1} the package is named docker-io and [docker](http://koji.fedoraproject.org/koji/buildinfo?buildID=442722) is something completely different.